Bottom: a6efdf7b585f84eb8578ed6014801ab3b8ef66bd
Top:    1bd50eced43ea4803993daaf85e33e117d6b0fa2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 16:10:42 +0100

hid/gtk/gtkhid-gl.c: Convert crosshair drawing to use PCB world coordinates


---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 31ee636..5fd8bb5 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -62,6 +62,9 @@
 #define	TO_PCB_UNITS(v)		(Settings.grid_units_mm ? MM_TO_COORD(v) : MIL_TO_COORD(v))
 
 extern int ghid_flip_x, ghid_flip_y;
+#define FLIP_X(x)   ((ghid_flip_x ? -(x) : (x)))
+#define FLIP_Y(y)   ((ghid_flip_y ? -(y) : (y)))
+
 #define SIDE_X(x)   ((ghid_flip_x ? PCB->MaxWidth - (x) : (x)))
 #define SIDE_Y(y)   ((ghid_flip_y ? PCB->MaxHeight - (y) : (y)))
